app-frontend/src/app/components/tools/toolCreateModal/toolCreateModal.controller.js

```diff
@@ -126,6 +126,84 @@ export default class ToolCreateModalController {
         return builtNode;
     }
 
+    expressionTreeToMAML(tree, value, allowCollapse = true) {
```

Comment:
I think this is from the other PR; something may have gone awry with the rebase.